src/pages/scripting/reference/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Reference
+description: Scripting Reference
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# Scripting Reference
+
+## Script module
+Represents the UXP module that contains the properties and methods used for scripting.
+
+### Usage
+---
+```js
+const script = await require("uxp").script;
+```
+
+### Properties
+
+---
+
+| Name | Type | Access | Description |
+| --- | --- | --- | --- |
+| args | Array[] | readOnly | Arguments passed while invoking the script |
+| executionContext | ExecutionContext | readOnly | ExecutionContext passed by the host when invoking Script |
+
+```jsx
+const args = script.args;
```

Comment:
Didn't we say above that this isn't ready yet?

Reply:
removed